packages/nimble-components/src/patterns/dropdown/types.ts

```diff
@@ -22,3 +23,17 @@ export const DropdownAppearance = {
 } as const;
 export type DropdownAppearance =
     (typeof DropdownAppearance)[keyof typeof DropdownAppearance];
+
+/**
+ * @internal
+ *
+ * This interface is used to register options with their parent once their
+ * 'connectedCallback' method is run. This allows for the "owner", like the
+ * Select, to have its value set to that newly registered option earlier than it
+ * might otherwise in certain situations. One such scenario is in an Angular
+ * reactive form, where the form value is set to an option immediately after
+ * dynamically adding it.
+ */
+export interface DropdownOwner {
```

Comment:
I think I'd go with `ListOptionOwner` since that's the type that the method in the interface takes.